pontoon/teams/views.py

```diff
@@ -168,41 +172,72 @@ def ajax_permissions(request, locale):
 
 @login_required(redirect_field_name='', login_url='/403')
 @require_POST
-def request_projects(request, locale):
-    """Request projects to be added to locale."""
-    slug_list = request.POST.getlist('projects[]')
-    locale = get_object_or_404(Locale, code=locale)
+def request_item(request, locale=None):
+    """Request projects and teams to be added."""
+    user = request.user
+    # Requesting a project
```

Comment:
Nit: Add blank line above this line and change the comment to "Request projects to be enabled for team".